pkg/get/get_test.go

```diff
@@ -470,7 +471,63 @@ func Test_DownloadInletsctl(t *testing.T) {
 			t.Fatal(err)
 		}
 		if got != tc.url {
-			t.Fatalf("want: %s, got: %s", tc.url, got)
+			t.Logf("want: %s, got: %s", tc.url, got)
+			t.Fail()
+		}
+	}
+}
+
+func Test_DownloadDoctl(t *testing.T) {
+	tools := MakeTools()
+	name := "doctl"
+
+	var tool *Tool
+	for _, target := range tools {
+		if name == target.Name {
+			tool = &target
+			break
+		}
+	}
+
+	type test struct {
+		os      string
+		arch    string
+		version string
+		url     string
+	}
+
+	const toolVersion = "1.46.0"
+	const urlPrefix = "https://github.com/digitalocean/doctl/releases/download"
+
+	tests := []test{
+		{os: "mingw64_nt-10.0-18362",
+			arch:    arch64bit,
+			version: toolVersion,
+			url:     urlPrefix + "/v" + toolVersion + "/doctl-1.46.0-windows-amd64.zip"},
+		{os: "linux",
+			arch:    arch64bit,
+			version: toolVersion,
+			url:     urlPrefix + "/v" + toolVersion + "/doctl-1.46.0-linux-amd64.tar.gz"},
+		{os: "darwin",
+			arch:    arch64bit,
+			version: toolVersion,
+			url:     urlPrefix + "/v" + toolVersion + "/doctl-1.46.0-darwin-amd64.tar.gz"},
+		// this asserts that we can build a URL for ARM processors, but no asset exists and will yield a 404
+		{os: "linux",
+			arch:    arm,
+			version: toolVersion,
+			url:     urlPrefix + "/v" + toolVersion + "/doctl-1.46.0-linux-.tar.gz"},
```

Comment:
@Waterdrips I'm not crazy about this, because anyone trying to install doctl with ARM will get a 404. Problem is DigitalOcean doesn't appear to have a binary that will work with ARM. Any recommendations on desired behavior here?

Reply:
@Waterdrips whatcha thinking Re: the ARM experience?